slybot/slybot/starturls/__init__.py

```diff
@@ -6,16 +6,16 @@
 import six
 from six.moves.urllib_parse import urlparse
 
+from .feed_generator import FeedGenerator
```

Comment:
So that we can have `from slybot.starturls import FeedGenerator`. The `__init__.py` file is used here to expose what we want from the python package.